.github/workflows/import_ac.yaml

```diff
@@ -0,0 +1,62 @@
+name: Import Android components
+on:
+  schedule:
+    - cron: "0 5 * * *"
```

Comment:
Is it intentional that all of the scripts run at the same time?

Reply:
Just for consistency (knowing when to expect notifications for new content), but we could probably spread them out within a few minutes.